.jenkins/actions/run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
The new approved way of writing `#!/bin/bash` is `#!/usr/bin/env bash`.

Reply:
Sure I can change that -- what's the difference? who approved it? :)